morecantile/models.py

```diff
@@ -342,57 +425,34 @@ def custom(
         """
         matrix_scale = matrix_scale or [1, 1]
 
-        tms: Dict[str, Any] = {
-            "title": title,
-            "id": id,
-            "crs": crs,
-            "tileMatrices": [],
-            "_geographic_crs": geographic_crs,
-        }
-
         if ordered_axes:
             is_inverted = ordered_axis_inverted(ordered_axes)
         else:
             is_inverted = crs_axis_inverted(crs)
 
-        if is_inverted:
-            tms["boundingBox"] = TMSBoundingBox(
-                crs=extent_crs or crs,
-                lowerCorner=[extent[1], extent[0]],
-                upperCorner=[extent[3], extent[2]],
-            )
-        else:
-            tms["boundingBox"] = TMSBoundingBox(
-                crs=extent_crs or crs,
-                lowerCorner=[extent[0], extent[1]],
-                upperCorner=[extent[2], extent[3]],
-            )
-
```

Comment:
removed `boundingBox` form the TileMatrixSet (now totally optional/useless in TMS 2.0)

Reply:
@roelarents You're right, there is an issue with our model, The `BoundingBox` only requires `lowerCorner, upperCorner`

https://github.com/opengeospatial/2D-Tile-Matrix-Set/blob/master/schemas/tms/2.0/json/2DBoundingBox.json

Would you be able to open an issue for this? 🙏